### ISSUE TITLE
Add Steam Deck OLED audio support

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#835]

**Comment by GuestSneezeOSDev** [comment]
may help https://wiki.archlinux.org/title/Steam_Deck#Audio

**Comment by LukeShortCloud** [comment]
Thanks @GuestSneezeOSDev , that is definitely the last piece of the puzzle we need. I created [linux-firmware-valve](https://aur.archlinux.org/packages/linux-firmware-valve) package which is step 1. Step 2 is to package the configuration files.

I found a `alsa-ucm-conf-1.2.9-1` package in both SteamOS Jupiter and in upstream Arch Linux. I grabbed the Arch Linux one from the Arch Linux Archive. The only difference is this one symlink:

- usr/share/alsa/ucm2/conf.d/acp5x/Valve-Jupiter-1.conf --> ../../AMD/acp5x/acp5x.conf

For `steamdeck-dsp`, it contains a lot of configuration files. We might be able to install it as-is.
